controllers/text_responses.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+import json
+
+
+# try something like
+def index(): return dict(message="hello from text_response.py")
+
+# Helper function to decode names with space in
+def decode_name(name):
```

Comment:
Be usre to set the type - such as:
```suggestion
def decode_name(name: str):
```

Reply:
Ok got it!

Will fix all of this when all the requests have been merged